docs/core/tools/dotnet-new-sdk-templates.md

```diff
@@ -567,154 +552,189 @@ The ability to create a project for an earlier TFM depends on having that versio
 
-  | SDK version | Default value   |
-  |-------------|-----------------|
-  | 9.0         | `net9.0`        |
-  | 8.0         | `net8.0`        |
-  | 7.0         | `net7.0`        |
+  | SDK version | Default value |
+  |-------------|---------------|
+  | 9.0         | `net9.0`      |
+  | 8.0         | `net8.0`      |
+  | 7.0         | `net7.0`      |
 
   To create a project that targets a framework earlier than the SDK that you're using, see [`--framework` for `console` projects](#template-options) earlier in this article.
 
 - **`--no-restore`**
 
   Doesn't execute an implicit restore during project creation.
 
-- **`--use-browserlink`**
+- **`--use-program-main`**
 
-  Includes BrowserLink in the project.
+  If specified, an explicit `Program` class and `Main` method will be used instead of top-level statements. Available since .NET SDK 6.0.300. Default value: `false`.
 
-- **`-rrc|--razor-runtime-compilation`**
+***
 
-  Determines if the project is configured to use [Razor runtime compilation](/aspnet/core/mvc/views/view-compilation#runtime-compilation) in Debug builds.
+### `apicontroller`
 
-- **`--kestrelHttpPort`**
+API Controller with or without read/write actions.
 
-  Port number to use for the HTTP endpoint in *launchSettings.json*.
+- **`-p:n|--name <NAME>`**
 
-- **`--kestrelHttpsPort`**
+  The namespace for the generated code. Default is `MyApp.Namespace`.
 
-  Port number to use for the HTTPS endpoint in *launchSettings.json*. This option is not applicable when the parameter `no-https` is used (but `no-https` is ignored when an individual or organizational authentication setting is chosen for `--auth`).
+- **`-ac|--actions`**
 
-- **`--use-program-main`**
+  Create a controller with read/write actions. Default is `false`.
 
-  If specified, an explicit `Program` class and `Main` method will be used instead of top-level statements. Available since .NET SDK 6.0.300. Default value: `false`.
+***
+
+### `globaljson`
+
+- **`--sdk-version <VERSION_NUMBER>`**
+
+  Specifies the version of the .NET SDK to use in the *global.json* file.
+
+- **`--roll-forward <ROLL_FORWARD_POLICY>`**
+
+  The roll-forward policy to use when selecting an SDK version, either as a fallback when a specific SDK version is missing or as a directive to use a later version.
+  For more information, see [global-json](global-json.md#rollforward).
+
+### `sln`
+
+Creates an empty solution file containing no projects.
+
+> [!NOTE]
+> In .NET SDK 9.0.200 and later, this template supports a `--format` option to choose between `sln` and `slnx` formats. Starting with .NET 10, the default format is `slnx`.
 
 ***
 
-## <a name="spa"></a> `angular`, `react`
+### `editorconfig`
+
+Creates an *.editorconfig* file for configuring code style preferences.
+
+- **`--empty`**
+
+  Creates an empty *.editorconfig* instead of the defaults for .NET.
+
+## Discontinued templates
```

Comment:
This is a great idea! I wondered how we'd be able to represent those older templates well. A separate section never occurred to me.